code/parachain/frame/composable-tests-helpers/src/lib.rs

```diff
@@ -1,2 +1,15 @@
 #![cfg_attr(not(feature = "std"), no_std)]
+
+use sp_runtime::AccountId32;
+
 pub mod test;
+
+pub const ALICE: AccountId32 = AccountId32::new([
```

Comment:
in xcm tests i used real keyring. would be easier to write simnode tests.

Reply:
not sure what you're referring to - what's keyring?

Reply:
sp_keyring

Reply:
It has well known accounts hardcoded, such as Alice and Bob

Reply:
yes, @PoisonPhang filled me in on that, cool stuff. I'm working on using that right now